### PR TITLE
Changes for version 0.0.3: script-based interface

### PR DESCRIPTION
**Pull request by christiam**

### CODE REVIEW

File: docs/source/issues.rst [hunk 1]
```diff
@@ -5,18 +5,16 @@ Known issues
...
+Batch length setting
+--------------------
 
 The value of :ref:`ELB_BATCH_LEN` greatly affects performance. We are in the
 process of determining reasonable defaults for various programs.
 
-.. _disk_leak:
+.. _suboptimal_performance:
 
-Persistent disk is not cleaned up
```
Comment (boratyng) [inline docs/source/issues.rst:16]: I would still leave a note about possible persistent disk leak, maybe saying that it is unlikely, but can happen. Especially if users keep on hitting Ctrl-C anything can happen.

File: docs/source/quickstart.rst [hunk 2]
```diff
@@ -64,14 +90,19 @@ Run the command below to download the results
...
+result files.
 
 .. code-block:: bash
 
-    make test_asn_results
+    find . -name "batch*.out.gz" -type f -print0 | \
+        xargs -0 -P8 -I{} -t gzip -t {}
+    find . -name "batch*.out.gz" -type f -print0 | \
+        xargs -0 -P8 -I{} -t bash -c "zcat {} |
+        datatool -m /netopt/ncbi_tools64/c++.metastable/src/objects/blast/blast.asn -M /am/ncbiapdata/asn/asn.all -v - -e /dev/null"
```
Comment (boratyng) [inline docs/source/quickstart.rst:105]: Does anyone outside of our group use BLAST archive format?

File: docs/source/support.rst [hunk 3]
```diff
@@ -3,9 +3,14 @@
 Support
 =======
 
-**N.B.**: Please help the development team help you: consider using the
-unix ``screen`` tool to capture the output of your usage of ElasticBLAST and
-attaching the session log to your problem report. Thanks! :)
+**N.B.**: Please help the development team help you:
```
Comment (boratyng) [inline docs/source/support.rst:6]: I like this. 😄

**Review by boratyng — COMMENTED** [review]
Is there a way to see this rendered?

**Comment by christiam** [comment]
> Is there a way to see this rendered?

Yes, click on "Details" on the checks ⬆️

File: docs/source/issues.rst (hunk 1 above)
Comment (christiam) [reply docs/source/issues.rst:16]: Sounds good, I added this in d1a7ae9.

File: docs/source/quickstart.rst (hunk 2 above)
Comment (christiam) [reply docs/source/quickstart.rst:105]: I don't know that. The benefit of using this format is that one can reformat in any other format without having to re-run BLAST. I added a link to blast_formatter in the documentation to highlight this: https://elbdoc--2.org.readthedocs.build/en/2/configuration.html#blast-options

**Review by boratyng — APPROVED** [review]

**Pull request merged**